lib/livebook/utils/time.ex

```diff
@@ -92,7 +97,7 @@ defmodule Livebook.Utils.Time do
 
   defp maybe_convert_to_minutes(duration), do: duration
 
-  defp duration_in_words({:seconds, seconds}) when seconds in 0..4 do
+  defp duration_in_words({:seconds, seconds}) when seconds <= 4 do
```

Comment:
Actually, we raise in `time_ago_in_words` above, so we need to remove the raise also!